deploy/snapshot/internal/controller/podsnapshotcontent_test.go

```
@@ -654,6 +673,38 @@ func TestRunCheckpoint_WritesFailedOnError(t *testing.T) {
 	assert.Equal(t, "CheckpointFailed", cond.Reason)
 }
 
+func TestRunCheckpoint_ReleaseFailureKillsTarget(t *testing.T) {
+	target := exec.Command("sleep", "30")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)